tools/cpp/unix_cc_toolchain_config.bzl

```diff
@@ -1181,6 +1181,16 @@ def _impl(ctx):
         ],
     )
 
+    treat_warnings_as_errors_feature = feature(
+        name = "treat_warnings_as_errors",
+        flag_sets = [
+            flag_set(
+                actions = [ACTION_NAMES.c_compile, ACTION_NAMES.cpp_compile],
+                flag_groups = [flag_group(flags = ["-Werror"])],
```

Comment:
I wonder if we should add `-Wl,-fatal-warnings` to link invocations here as well?

Reply:
According to [this](http://web.mit.edu/rhel-doc/3/rhel-as-en-3/w.html), `-fatal-warnings` is an argument for the GNU assembler (`as`). So I should add `-fatal-warnings` for assemble actions?

Reply:
checkout https://linux.die.net/man/1/ld

Reply:
Thanks, I have added `-Wl,-fatal-warnings` for link invocations